src/main/resources/application-h2.properties

```diff
@@ -21,6 +21,9 @@ spring.jpa.hibernate.use-new-id-generator-mappings=false
 spring.jpa.hibernate.ddl-auto=none
 spring.jpa.defer-datasource-initialization=true
 
+# HikariCP configuration
```

Comment:
not needed [connectionTimeout](https://github.com/brettwooldridge/HikariCP#frequently-used) sets to 30s
_This property controls the maximum number of milliseconds that a client (that's you) will wait for a connection from the pool. If this time is exceeded without a connection becoming available, a SQLException will be thrown. Lowest acceptable connection timeout is 250 ms. Default: 30000 (30 seconds)_

Reply:
yes, it is a default parameter, I set it only as reminder to highlight difference with the other properties files.